### PR TITLE
TissueSample(Collection) updates

### PR DESCRIPTION
**Pull request by UlrikeS91**
Intro of "descendedFrom" to TissueSample and TissueSampleCollection (pointing to all specimen states as options).

### CODE REVIEW

File: schemas/research/tissueSample.schema.tpl.json [hunk 1]
```diff
@@ -6,7 +6,19 @@
     "studiedState",
     "type"
   ],
-  "properties": {
+  "properties": {    
+    "descendedFrom": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add all specimen states used to produce or obtain this tissue sample from.",
```
Comment (lzehl) [inline schemas/research/tissueSample.schema.tpl.json]: delete "from"

File: schemas/research/tissueSampleCollection.schema.tpl.json [hunk 2]
```diff
@@ -19,6 +19,18 @@
         "https://openminds.ebrains.eu/sands/ParcellationEntityVersion"
       ]
     },    
+    "descendedFrom": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add all specimen states used to produce or obtain this tissue sample collection from.",
```
Comment (lzehl) [inline schemas/research/tissueSampleCollection.schema.tpl.json]: delete "from"

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/research/tissueSample.schema.tpl.json (hunk 1 above)
Comment (UlrikeS91) [reply schemas/research/tissueSample.schema.tpl.json]: done

File: schemas/research/tissueSampleCollection.schema.tpl.json (hunk 2 above)
Comment (UlrikeS91) [reply schemas/research/tissueSampleCollection.schema.tpl.json]: done

**Review by lzehl — APPROVED** [review]

**Pull request merged**